frontend/src/types/media.ts

```diff
@@ -122,3 +122,10 @@ export const isMediaDetail = <T extends SupportedMediaType>(
 ): media is DetailFromMediaType<T> => {
   return !!media && media.frontendMediaType === mediaType
 }
+
+export type Metadata = {
+  label: string
```

Comment:
It seems as if `label` is always passed through `i18n.t()` so we could simplify the `metadata.ts` file if we move these calls to the `VMetadata.vue` component inside `v-for`.